apps/nova/src/components/TutorProfile/RateForm.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useCallback } from "react";
+import { useEditRateTutor, useRateTutor } from "@litespace/headless/rating";
+import {
+  Button,
+  ButtonSize,
+  ButtonType,
+  Controller,
+  Field,
+  Form,
+  Label,
+  toaster,
+  useFormatMessage,
+} from "@litespace/luna";
+import { useForm } from "react-hook-form";
+import { IRating, Void } from "@litespace/types";
+import { useQueryClient } from "@tanstack/react-query";
+
+type IForm = {
+  feedback?: string;
+  rating: number;
+};
+
+type RateFormProps = {
+  tutor: number;
+  rate?: IRating.Populated;
+  close?: Void;
+};
+
+const RateForm: React.FC<RateFormProps> = ({ tutor, rate, close }) => {
+  const intl = useFormatMessage();
+  const queryClient = useQueryClient();
+  const onSuccess = useCallback(() => {
+    queryClient.invalidateQueries({ queryKey: ["tutor-rating", tutor] });
```

Comment:
Please use the new hook `useInvalidate`

Reply:
I added that hook in another pr, do i need to still do this change?

Reply:
no, we can do it later.